# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../e-utils/ecore_libs.patch
# Copyright (C) 2007 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- e_utils/src/bin/e17setroot/Makefile.am.orig	2007-06-28 17:32:22.000000000 +0200
+++ e_utils/src/bin/e17setroot/Makefile.am	2007-06-28 17:32:46.000000000 +0200
@@ -15,6 +15,6 @@
 e17setroot_SOURCES = \
 e17setroot.c
 
-e17setroot_LDADD = @EWL_LIBS@ @EET_LIBS@ @ENGRAVE_LIBS@ @IMLIB2_LIBS@ @ENLIGHTENMENT_LIBS@
+e17setroot_LDADD = @EWL_LIBS@ @EET_LIBS@ @ENGRAVE_LIBS@ @IMLIB2_LIBS@ @ENLIGHTENMENT_LIBS@ @ECORE_LIBS@
 
 
